packages/shopware-6-client/src/interfaces/models/common/Collection.ts

```diff
@@ -0,0 +1,3 @@
+export interface Collection {
+  elements: [];
```

Comment:
please provide an example of this collection usage

also, we need to add a type of elements array

Reply:
We can delete this one as we deleted all the interfaces suffixed with Collection.